frontend/sw360-portlet/src/main/webapp/html/include/searchfield.jspf

```diff
@@ -0,0 +1,89 @@
+<%--
```

Comment:
I've put the search field in a jspf because this component also needs some html code.